### ISSUE TITLE
Fix Request: False positive in AvoidObjectMapperAsField

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#230]
no violation if the objectMapper is used as in jaxMsgConverter.setObjectMapper(objectMapper)
```
import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
import com.fasterxml.jackson.databind.ObjectMapper;

class Foo {
    private static final ObjectMapper objectMapper = new ObjectMapper();
    void bar() {
        final MappingJackson2HttpMessageConverter jaxMsgConverter = new MappingJackson2HttpMessageConverter();
        jaxMsgConverter.setObjectMapper(objectMapper);
    }
}
```
